packages/php-config-printer/src/ServiceOptionConverter/FactoryConfiguratorServiceOptionKeyYamlToPhpFactory.php

```diff
@@ -24,7 +24,9 @@ public function decorateServiceMethodCall(
         mixed $values,
         MethodCall $methodCall
     ): MethodCall {
-        $args = $this->argsNodeFactory->createFromValuesAndWrapInArray($yaml);
+        $args = (is_array($yaml) || strpos($yaml, ':') !== false)
```

Comment:
Should condition be extracted to some private method? Or maybe you don't want ternary here at all?